nginx.conf

```diff
@@ -0,0 +1,49 @@
+http {
+    # Replace nginx.conf in "/etc/nginx/" with this file config
+    # Run nginx with "nginx"
+    # Stop nginx with "nginx -s stop"
+    # Reload nginx without stopping server "nginx -s reload"
+
```

Comment:
This all looks good! I would add error logging and access logging while you're at it. https://linuxize.com/post/nginx-log-files/